src/saleor_app/webhook.py

```diff
@@ -66,6 +67,7 @@ def decorator(func: WebHookHandlerSignature):
                 dependencies=[
                     Depends(verify_saleor_domain),
                     Depends(verify_webhook_signature),
+                    Depends(verify_issuing_principal_type)
```

Comment:
As stated above, this should not be enabled globally.

Reply:
Done